keyboards/handwired/footy/keymaps/default/keymap.json

```
@@ -0,0 +1,10 @@
+{
+    "keyboard": "handwired/footy",
+    "keymap": "default",
+    "layout": "LAYOUT",
+    "layers": [
+       [
+        "KC_A", "KC_B", "KC_C"
+       ]
+    ]
+}
```

Proper file ending. Or `qmk format-json` will do this.
```suggestion
}

```